docs/source/data-structures/presentations/examples.rst

```diff
@@ -75,6 +77,7 @@ documented in the :ref:`default-presentations` section.
 .. autosummary::
     :signatures: short
 
+    abacus_jones_monoid_AJP25
```

Comment:
There should probably also be a `braid_group_Bir05`, but that's an issue for `libsemigruops`